### PR TITLE
Revocation rate by county of residence

### PR DESCRIPTION
**Pull request by jjdressel11**
Updates the Revocation by County chart to be Revocation Rate by County of Residence.

### CODE REVIEW

File: server/core/demo_data/revocation_rate_by_county_60_days.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
+{"state_code":"US_DEMO","county_code":"US_DEMO_MCLEAN","revocation_count":"3","population_count":"64","revocation_rate":0.015625}
```
Comment (jessex) [inline server/core/demo_data/revocation_rate_by_county_60_days.json]: All of these are fake fixtures, yes?

File: src/components/charts/revocations/RevocationRateByCounty.js [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+    return revocationRate.toFixed(2);
+  }
+
+  return 0;
+}
+
+function valueCountsForCounty(officeValues, countyName) {
+  const officeNumbers = officeValues[countyName.toUpperCase()];
+  if (officeNumbers) {
+    return officeNumbers.revocationCount.concat('/', officeNumbers.populationCount);
```
Comment (jessex) [inline src/components/charts/revocations/RevocationRateByCounty.js]: Can we use the more standard format of `${officeNumbers.revocationCount}/${officeNumbers.populationCount}`?

File: src/components/charts/revocations/RevocationRateByCounty.js [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+  }
+
+  return '0/0';
+}
+
+function colorForCounty(chartDataPoints, countyName, maxValue, useDark) {
```
Comment (jessex) [inline src/components/charts/revocations/RevocationRateByCounty.js]: A couple of these functions look identical to the ones in the `RevocationsByCounty` chart component. Can we move those out to some set of, say, county choropleth utilities?

File: server/core/demo_data/revocation_rate_by_county_60_days.json (hunk 1 above)
Comment (jjdressel11) [reply server/core/demo_data/revocation_rate_by_county_60_days.json]: Yep

**Comment by jjdressel11** [comment]
> Looks good, @jjdressel11! I know we were discussing the headline format for this chart but it looks like we haven't added a headline yet. Is that to come later?

Double checked with Clem and she agreed that not adding a headline is okay for now. It's not clear what the header should be for this chart because the county with the highest rate is often a county with a very low supervision population - which is not as important to highlight in a takeaway header.

File: src/components/charts/revocations/RevocationRateByCounty.js (hunk 3 above)
Comment (jjdressel11) [reply src/components/charts/revocations/RevocationRateByCounty.js]: ugh fine

**Pull request merged**